lib/llm/conversation.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module LLM
+  class Conversation
+    attr_accessor :thread
```

Comment:
This should be documented.
Should it also be `attr_reader :thread` instead ?

Reply:
@0x1eef Indeed! That was a blindspot